onnxruntime/test/optimizer/qdq_transformer_test.cc

```diff
@@ -1922,6 +1999,7 @@ TEST(QDQTransformerTests, Clip) {
       auto* clip_output = builder.MakeIntermediate();
       constexpr float min = .0f;
       constexpr float max = 6.0f;
+      auto opset_version = builder.DomainToVersionMap().find("")->second;
```

Comment:
There's an `opset_version` argument in the test case lambda so not clear why we need to do the lookup here.

Reply:
When would the values differ? Doesn't TransformerTester use opset_version to create the domain to version map in the builder?

Reply:
It should not differ. I'm just using the same code as I did in all the other unit tests I modified.